src/task4/index.ts

```diff
@@ -0,0 +1,45 @@
+import express from "express";
+import { sequelize } from "./data-access/db";
+
+import ValidationService from "./services/ValidationService";
+import {
+  getUsers,
+  getUser,
+  addUser,
+  updateUser,
+  getAutoSuggestList,
+  deleteUser
+} from "./routers/Users";
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+export const app = express();
+
+app.listen(3000);
```

Comment:
Good - just please review the `loaders` part of https://softwareontheroad.com/ideal-nodejs-project-structure/#folder , this is useful patter - in prod apps index fill will grow usually to enormous sizes if you put all initialization logic here. So try to use loaders